src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/ResourceLocator.js

```diff
@@ -7,6 +7,8 @@ import ResourceLocatorHistory from '../../../containers/ResourceLocatorHistory';
 import Requester from '../../../services/Requester';
 import type {FieldTypeProps} from '../../../types';
 import resourceLocatorStyles from './resourceLocator.scss';
+import {translate} from "../../../utils/Translator";
+import Button from "../../../components/Button";
```

Comment:
Regarding the two previous lines:
- We use single quote instead of double quotes
- The imports using `..` should be before imports with `.`

These are probably some of the reasons CI is currently failing.